sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/api/Microsoft.Azure.WebJobs.Extensions.EventHubs.netstandard2.0.cs

```diff
@@ -17,6 +17,10 @@ public sealed partial class EventHubTriggerAttribute : System.Attribute
         public string ConsumerGroup { get { throw null; } set { } }
         public string EventHubName { get { throw null; } }
     }
+    public static partial class IAsyncCollectorExtensions
+    {
+        public static System.Threading.Tasks.Task AddAsync(this Microsoft.Azure.WebJobs.IAsyncCollector<Azure.Messaging.EventHubs.EventData> instance, Azure.Messaging.EventHubs.EventData item, string partitionKey, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
nit: should we call the param eventData to match the EventDataBatch from the library?

Reply:
I called it "item" to match the existing `IAsyncCollector<T>` nomenclature.

Reply:
Updated.